tests/fib/test_fib.py

```diff
@@ -170,6 +170,52 @@ def filter_ports(all_port_indices, tbinfo):
     return host_ptf_ports_all
 
 
+def get_port_and_portchannel_members(port_name, all_port_indices, duts_minigraph_facts,
+                                     upstream_lc, tbinfo):
+    """
+    Get PTF port indices for a port and all its port channel members (if applicable).
+
+    Args:
+        port_name: The DUT port name to check
+        all_port_indices: Dictionary mapping PTF port indices to (asic_id, port_name)
+        duts_minigraph_facts: Minigraph facts containing port channel information
+        upstream_lc: The upstream line card hostname
+
+    Returns:
+        List of PTF port indices for the port and all its port channel members
+    """
+
+    # for T2(except UT2) topologies, no need to append, as we are already filtering out the whole upstream lc ports
+    if tbinfo['topo']['type'] == 't2' and 't2_single_node' not in tbinfo['topo']['name']:
+        return []
+
+    mg_facts = None
+    for asic_id, asic_data in duts_minigraph_facts[upstream_lc]:
+        mg_facts = asic_data
+        break  # Use first ASIC's facts for port channel lookup
```

Comment:
If `port_name` is on a different ASIC than the mg_facts will we still have it's port_channel information in mg_facts?